Backend/filesystem/service.go

```diff
@@ -92,3 +92,31 @@ func deleteEntity(pool database.Pool, entityID int) error {
 	}
 	return nil
 }
+
+// renameEntity changes the logical name of a given object in the filesystem
+func renameEntity(pool database.Pool, entityID int, newName string) error {
+	_, err := pool.GetConn().Exec(context.Background(),
+		"UPDATE filesystem SET logicalname = ($1) WHERE entityid = ($2)", newName, entityID)
+	if err != nil {
```

Comment:
can just:
```go
return err
```

Reply:
can you also update: https://github.com/csesoc/cms.csesoc.unsw.edu.au/blob/prototype/Backend/filesystem/service.go#L87 when you do that? I realised i did the same thing lmao

Reply:
yup - only did this because you did it too